tests/mahalanobis_distance/mahalanobis_test.py

```diff
@@ -0,0 +1,9 @@
+from pandas import read_csv
+from numpy import mean, cov
+from redata.statistics.mahalonobis_distance import MahalanobisDistance
+
```

Comment:
just to make sure it's not run when  importing this file, can you add `if __name__ == "__main__": to this python script